.gitignore

```diff
@@ -29,3 +29,9 @@ env
 env3
 venv
 venv3
+
+# local files
```

Comment:
Ideally this wouldn't be here (.gitignore shouldn't really have personal file locations) but we can leave it for now